config/initializers/devise.rb

```diff
@@ -294,6 +294,12 @@
 
   config.jwt do |jwt|
     jwt.secret = ENV['DEVISE_JWT_SECRET_KEY']
+    jwt.dispatch_requests = [
```

Comment:
I think we can remove these, since we figured out the problem was that CORS issue?